packages/airnode-node/src/workers/local-gateways/validation.test.ts

```diff
@@ -109,6 +115,282 @@ describe('verifyHttpSignedDataRequest', () => {
   });
 });
 
+describe('dropInvalidBeacons', () => {
+  const validBeacon = {
+    // beaconId: 0x1032c3cbea7692429f3f1bdb72c47b5c61bdd3ca995a763027f8aa511b42b11b
+    // templateId: 0x64a8f8e70cd1bd4e4621bde25053bf4e22633241effa9f768bf18ff6400dc702
+    airnodeAddress: '0x9A2Df85E73851e27044504d72563696E5cE86B95',
+    endpointId: '0xa473a7ca2d5211e6e5766cc6a27c6e90a4f0270f13565e303c56a629815ed60a',
+    encodedParameters:
+      '0x3173000000000000000000000000000000000000000000000000000000000000636f696e49640000000000000000000000000000000000000000000000000000657468657265756d000000000000000000000000000000000000000000000000',
+    timestamp: '1677747253',
+    encodedValue: '0x0000000000000000000000000000000000000000000000000000000000000064', // 100
+    signature:
+      '0x9122514d1cb4598435ea21afb7790d9daf5850b87673872b7ddf9e8df7a6afb3106a9f93557429b2b8bca4c9c57e103b93ab0de11f7bec8feeca1968189bbd8f1b',
+  };
+  const validDecodedBeacon = { ...validBeacon, decodedValue: ethers.BigNumber.from(100) };
+
+  const currentTimestamp = 1677790659;
+  beforeAll(() => {
+    jest.useFakeTimers();
+    jest.setSystemTime(currentTimestamp);
+  });
+
+  afterAll(() => {
+    jest.useRealTimers();
+  });
+
+  it('drops beacons without data', () => {
+    const missingDataBeacon = omit(validBeacon, 'signature');
```

Comment:
That's probably a poor description. What I meant was that if one of the "data" fields (`signature`, `encodedValue` or `timestamp`) is missing we've obtained a beacon without "data". That means that it's there only so that we know how many beacons are in that particular beacon set.

Reply:
Yeah, I thought so after reading the other tests. I guess just edit the test title.